modules/memfault-firmware-sdk/Kconfig

```
@@ -153,6 +153,7 @@ config MEMFAULT_NCS_LTE_METRICS
 	bool "Collect LTE metrics"
 	select LTE_LC_TRACE
 	depends on LTE_LINK_CONTROL
+	depends on MODEM_INFO
```

Nit: We can combine these into a single statement with &&
```suggestion
	depends on LTE_LINK_CONTROL && MODEM_INFO
```

---

I like it! Done